code/game/machinery/vending.dm

```diff
@@ -371,17 +371,16 @@
 				display_records = product_records + coin_records
 			if((coin || bill) && extended_inventory)
 				display_records = product_records + hidden_records + coin_records
-			dat += "<ul>"
+			dat += "<table>"
 			for (var/datum/data/vending_product/R in display_records)
-				dat += "<li>"
+				dat += "<tr><td>" + GetIconForProduct(R) + "</td>"
+				dat += "<td style=\"width: 100%\"><b>[sanitize(R.product_name)]</b></td>"
 				if(R.amount > 0)
-					dat += "<a href='byond://?src=\ref[src];vend=\ref[R]'>Vend</a> "
+					dat += "<td><b>[R.amount]&nbsp;</b></td><td><a href='byond://?src=\ref[src];vend=\ref[R]'>Vend</a></td>"
 				else
-					dat += "<span class='linkOff'>Sold out</span> "
-				dat += "<font color = '[R.display_color]'><b>[sanitize(R.product_name)]</b>:</font>"

```

Comment:
You removed what I think was the only usage of `var/display_color`. Axe it as well while you're at it, please